Source/Core/Core/PowerPC/Jit64/Jit.cpp

```diff
@@ -967,8 +966,7 @@ bool Jit64::DoJit(u32 em_address, JitBlock* b, u32 nextPC)
     {
       // Gather pipe writes using a non-immediate address are discovered by profiling.
       const u32 prev_address = m_code_buffer[i - 1].address;
-      bool gatherPipeIntCheck =
-          js.fifoWriteAddresses.find(prev_address) != js.fifoWriteAddresses.end();
+      bool gatherPipeIntCheck = js.fifoWriteAddresses.contains(prev_address);
```

Comment:
There is an identical find != end statement in JitArm64, but it seems your automated tool didn't consider source files not compiled on your platform. Better to change it in both places for consistency.

Reply:
Which file is that? I can't find any JitArm64 without some suffix.

Reply:
It is a file also called Jit.cpp, but located in the JitArm64 directory rather than Jit64.